Generate.fs

```diff
@@ -29,11 +29,7 @@ type CheckAction =
     }
 
 [<CLIMutable>]
-type CodebasePrepStep =
-    {
-        Command : string
-        Args : string
-    }
+type CodebasePrepStep = { Command : string ; Args : string }
```

Comment:
You could put `fsharp_max_record_width = 0` in your `.editorconfig`.
This will use the long format if the length of the record is more than 0 characters.